src/app/account/invitation/[inviteId]/page.tsx

```diff
@@ -11,18 +10,16 @@ export default async function AcceptInvitePage({ params }: { params: Promise<{ i
 
     const invite = await db
         .selectFrom('pendingUser')
-        .select('email')
+        .select(['email', 'orgId']) // Fetch orgId to potentially get orgName later if needed
```

Comment:
this should fetch the org and other information needed by InviteHandler so that it doesn't need to call any actions to fetch the data.
```suggestion
        .join('org', 'pendingUser.orgId', 'org.id')
        .select(['email', 'orgId', 'org.name as orgName', 'org.slug as orgSlug'])
```